src/test/kotlin/org/rustSlowTests/cargo/runconfig/RunConfigurationTestBase.kt

```diff
@@ -97,6 +93,12 @@ abstract class RunConfigurationTestBase : RsWithToolchainTestBase() {
         return CargoBuildManager.build(buildConfiguration).get()
     }
 
+    protected fun checkProject(useClippy: Boolean = false): CargoBuildResult {
+        val checkConfiguration = createBuildConfiguration()
+        checkConfiguration.configuration.command = if (useClippy) "clippy" else "check"
```

Comment:
Let's avoid such hacks. Here you create build configuration for default cargo command (`run` I suppose) and only after that switch actual cargo command. As a result, you didn't check your changes in `CargoBuildManager`. Try to revert them and run the corresponding tests, they will pass.

I suppose the right way to do it is to create `CargoCommandConfiguration` with proper command, for example, pass desired command to `org.rustSlowTests.cargo.runconfig.RunConfigurationTestBase#createConfiguration` (it may be also useful in `RsCoverageTest`)

Reply:
BTW currently we don't have tests for `build` command itself, only for `run`